frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {
+    Card, CardTitle, CardText, CardDeck,
+    CardBody, CardSubtitle
+  } from 'reactstrap';
+  
+function Features() {
+    return (
+        <div>
+            <h1 className="text-center">Features of our Application</h1>
+            
+            <CardDeck>
```

Comment:
What made you go with a CardDeck? I have never used it myself, just wondering what led you to using it. Not a critique just asking out of curiosity.

Reply:
I wanted to use several cards for the features section as that's normally how I see features for something listed on a website. I was going to use a CardGroup but CardGroup has all the cards stuck together, and I wanted the cards to be each separate entity, and CardDeck does that.